aiven/client/cliarg.py

```diff
@@ -102,6 +102,9 @@ def wrapped(self):
     help="Additional configuration option whose value is loaded from file in the form name=filename",
 )
 arg.country_code = arg("--country-code", help="Billing country code")
+arg.disk_space_mb = arg(
+    "--disk-space-gib", dest="disk_space_mb", type=lambda value: int(value) * 1024, help="Disk space for data storage (GiB)"
```

Comment:
do we want to have some kind of sanity check here ? (eg: -ve value, super large value...)
or does the API do that ?

Reply:
Yes, API checks that the value is between the limits set by the plan.